Sprint-1/explore/chrome.md

```diff
@@ -10,9 +10,15 @@ Let's try an example.
 In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
-What effect does calling the `alert` function have?
+What effect does calling the `alert` function have?  
+console: alert('Hello World')
+Answer: Displays a dialog box bearing Hello World.
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
-What effect does calling the `prompt` function have?
-What is the return value of `prompt`?
+What effect does calling the `prompt` function have? 
+console: myName = prompt('What is your name')
+Answer: Display an input text box with a label What is your name
+
+What is the return value of `prompt`? 
+Answer: The value provided to the input box. In my own case, my first name, Kazeem
```

Comment:
What would be the return value if a user does not enter any value and simply clicked "OK"?
What would be the return value if a user entered some value but clicked "Cancel" instead of "OK"?

Reply:
> What would be the return value if a user does not enter any value and simply clicked "OK"? What would be the return value if a user entered some value but clicked "Cancel" instead of "OK"?

Answer 1: The return value is a single space character. ' '
Answer 2: The return value is null.